src/Sql.fs

```diff
@@ -7,6 +7,14 @@ open System.Data
 open System.Collections.Generic
 open FSharp.Control.Tasks
 
+open System.Reflection
+open Microsoft.FSharp.Reflection
+
+module internal Utils =
+    let isOption (p:PropertyInfo) = 
+        p.PropertyType.IsGenericType &&
+        p.PropertyType.GetGenericTypeDefinition() = typedefof<Option<_>>
```

Comment:
The file slowly but surely gets too long. I think this one should go into a separate file, but I intend to spend some time later on to refactor this, so for now just putting the module here.